apps/web/src/pages/test/constants.ts

```diff
@@ -112,13 +108,18 @@ export const QUIZ_QUESTIONS: QuizQuestion[] = [
       {
         id: "A",
         text: "와..사람 잘못봤네.. 민초파였어?",
-        isCorrect: true,
       },
       {
         id: "B",
         text: "당신은 훈이머리를 좋아하십니까?",
-        isCorrect: false,
       },
     ],
+    answer: "A",
   },
 ];
+
+export const SCORES = {
+  GOLD_SCORE: 6,
+  SILVER_SCORE: 3,
+  BRONZE_SCORE: 1,
+};
```

Comment:
SCORE라는 맥락은 이미 상수명에서 제공해주니까 제외해도 될것 같아~

```suggestion
export const SCORES = {
  GOLD: 6,
  SILVER: 3,
  BRONZE: 1,
};
```